src/main/java/pl/byczazagroda/trackexpensesappbackend/service/CustomUserDetails.java

```diff
@@ -0,0 +1,71 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import pl.byczazagroda.trackexpensesappbackend.model.User;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+
+public class CustomUserDetails implements UserDetails {
+    private final User user;
```

Comment:
One empty line before this field
```suggestion
public class CustomUserDetails implements UserDetails {

    private final User user;
```